doc/getting_started/starting.rst

```diff
@@ -10,10 +10,11 @@ Basic Statistical Visualization
 
 This tutorial will guide you through the basic process of creating
 visualizations in Altair. First, you will need to make sure you have the Altair
-package and its dependencies installed (see :ref:`installation`) and make sure
-you understand how altair plots are displayed (see :ref:`displaying-charts`).
+package and its dependencies installed (see :ref:`installation`).
 This tutorial will assume you are working within a Jupyter notebook user
-interface, so that plots are automatically rendered.
+interface, so that plots are automatically rendered.  If you are using another
+backend such as JupyterLab or VSCode, you may want to read about how Altair 
+plots are displayed before proceeding (see :ref:`displaying-charts`).
```

Comment:
```suggestion
interface (such as JupyterLab or VS Code), so that plots are automatically rendered. 
If you are using another interface, you may want to read about how Altair 
plots are displayed before proceeding (see :ref:`displaying-charts`).
```